src/main/java/at/hannibal2/skyhanni/config/features/chat/ChatConfig.java

```diff
@@ -142,4 +143,27 @@ public String toString() {
     @ConfigEditorBoolean
     @FeatureToggle
     public boolean petRarityDropMessage = true;
+
+    @Expose
+    @ConfigOption(name = "Stash Warnings", desc = "Compact or hide warnings relating to items/materials in your stash.")
+    @ConfigEditorDropdown
+    public StashHandlerType stashWarnings = StashHandlerType.COMPACT;
+
+    public enum StashHandlerType {
+        NONE("None (Show all)"),
```

Comment:
Maybe could indicate somehow that this means showing the default hypixel stash message